modules/manager/controller_setup.go

```diff
@@ -145,11 +146,24 @@ func (c *ControllerDef) MaybeSetupWithManager(ctx context.Context, mgr ctrl.Mana
 	return c.Controller.SetupWithManager(ctx, mgr)
 }
 
-func setupIndexes(ctx context.Context, mgr manager.Manager, cfg Config) error {
+func setupIndexes(ctx context.Context, logger logr.Logger, mgr manager.Manager, cfg Config) error {
 	if cfg.ControlPlaneControllerEnabled || cfg.GatewayControllerEnabled {
-		if err := index.DataPlaneNameOnControlPlane(ctx, mgr.GetCache()); err != nil {
+		log.GetLogger(ctx, "ControlPlane", cfg.DevelopmentMode).Info(
```

Comment:
Why not use the logger here (and below) which has been added to arguments?

Reply:
Those loggers as arguments are a leftover from my previous tries, I'm doing it this way to have the same format of log for indices as for those konnect one https://github.com/Kong/gateway-operator/blob/8c5a99a2c043c164bb2bc51489c1344727ca62b5/modules/manager/controller_setup.go#L570-L576